remote-config/src/test/kotlin/com/rakuten/tech/mobile/remoteconfig/ConfigCacheSpec.kt

```diff
@@ -125,11 +126,45 @@ class ConfigCacheSpec : RobolectricBaseSpec() {
 
         When calling stubVerifier.verify(any()) itReturns false
 
-        val cache = ConfigCache(stubFetcher, createFile(filename), stubPoller, stubVerifier)
+        val cache = ConfigCache(stubFetcher, createFile(filename), stubPoller, stubVerifier, false)
 
         cache["foo"] shouldEqual null
     }
 
+    @Test
+    fun `should not apply after fetching`() {
+        val filename = "cache.json"
+
+        When calling stubFetcher.fetch() itReturns createConfig(hashMapOf("foo" to "bar"))
+        When calling stubVerifier.verify(any()) itReturns true
+        val cache = ConfigCache(stubFetcher, createFile(filename), AsyncPoller(1), stubVerifier, false)
+
+        CoroutineScope(Dispatchers.Main).launch {
```

Comment:
I think you should be able to run this test using `runBlockingTest` and then you don't need to create your own scope: https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-blocking-test.html

Reply:
used `runBlockingTest`